### ISSUE TITLE
Add library to a public repository

### ISSUE THREAD

**Issue by pushbit** [tumblr/Bookends#2]
For example, Maven Central or JCenter.

**Comment by mlapadula** [comment]
Ah, working on getting access to Maven Central -- meanwhile, you can include it as a dependency via jitpack: https://jitpack.io/#tumblr/Bookends/dd5c17ec6af035e2e513e6454a9b370e94fbba1e

**Comment by jitpack-io** [comment]
@mlapadula Would be nice to have github release as well:)

**Comment by KevinTCoughlin** [comment]
Github release done & instructions added in https://github.com/tumblr/Bookends/commit/ded4bd921071224982b14a8db8098f99ca7b40ba

**Comment by pushbit** [comment]
Are there still plans to make the library available via Maven Central?

**Comment by mlapadula** [comment]
Maybe I closed this prematurely, I've uploaded to Bintray and am waiting on approval to be included in JCenter. If you want to check it out the project exists at: https://bintray.com/mlapadula/Bookends/Bookends/view

**Comment by mlapadula** [comment]
Now in JCenter :D
